src/server/graphql/models/Meeting/meetingMutation.js

```diff
@@ -0,0 +1,81 @@
+import {Meeting} from './meetingSchema';
+import r from '../../../database/rethinkdriver';
+import {isLoggedIn} from '../authorization';
+import uuid from 'node-uuid';
+import {
+  GraphQLBoolean,
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLID,
+  GraphQLList
+} from 'graphql';
+
+export default {
+  createMeeting: {
+    type: Meeting,
+    async resolve(source, args, {rootValue}) {
+      const {authToken} = rootValue;
+      // isLoggedIn(authToken);
+      const newMeeting = {
+        // TODO: a uuid is overkill. let's make it small for smaller urls & friendly socket payloads
+        id: uuid.v4(),
```

Comment:
NB: as a matter of style, yes uuids are overkill. Starting with them does make horizontal scaling 'safer'/easier.